dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/management/JPAPUnitInfo.java

```diff
@@ -497,7 +501,7 @@ private void addMappingFileNames(List<String> newValues) {
      * environment.
      *
      * @param jarFileValues List of jar file paths from <jar-file> in persistence.xml
-     * @param looseConfig class holding loose config mappings
+     * @param looseConfig   class holding loose config mappings
```

Comment:
It seems your editor is not properly honoring the .settings for this project; which are the default source formatting for open-liberty.  Your editor seems to have  inadvertantly introduced a bunch of extra white space that will be stripped out next time someone edits this file.  Would be nice if you could use an editor that doens't introduce extra whitespace, and avoid unnecessary churn across multiple changes.

Reply:
I'm using regular Eclipse, which should be pulling in the editor .settings.